docs/build.md

```diff
@@ -0,0 +1,96 @@
+### Dependencies for Installing/Building from Source:
```

Comment:
A better name would be INSTALL.md or BUILD.md. Another question: What do you think of putting this file in the root repo instead of in `docs`?